Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>7.3.0-preview</VersionPrefix>
+    <VersionPrefix>7.5.0-preview</VersionPrefix>
```

Comment:
7.5?

Reply:
this should always be at least one version ahead.  if we publish 7.4 then this needs to be at least 7.5.  as far as i'm concerned we could just set it to 7.99.0-preview and leave it be.  The only reason it needs to be ahead of the nuget version at all is when referencing a dependency that depends on this package, such as when referencing GraphQL Server 7.3.0 but trying to use a GraphQL.NET preview.  If the GraphQL.NET preview number was 7.2.0-preview then nuget would attempt to bump the nuget package of GraphQL.NET when restoring packages.  But if GraphQL.NET was 7.99.0-preview then there would never be a problem.

Reply:
I would switch to MinVer to manage version automatically.

Reply:
Yeah, I'm not a fan of adding more build-time dependencies when they are not necessary.  Our system works perfectly now and it took me longer to write these posts than any number of version bumps over the last couple years.